sdk/containerregistry/container-registry/review/container-registry.api.md

```diff
@@ -35,19 +31,132 @@ export class ContainerRegistryUserCredential {
     }
 
 // @public
-export type ContentProperties = ChangeableAttributes;
+export class ContainerRepositoryClient {
+    constructor(endpointUrl: string, repository: string, credential: TokenCredential | ContainerRegistryUserCredential, options?: ContainerRegistryClientOptions);
+    delete(options?: DeleteOptions): Promise<DeleteRepositoryResult>;
+    deleteRegistryArtifact(digest: string, options?: DeleteRegistryArtifactOptions): Promise<RestResponse>;
+    deleteTag(tag: string, options?: DeleteTagOptions): Promise<RestResponse>;
+    endpoint: string;
+    getProperties(options?: GetPropertiesOptions): Promise<RepositoryProperties>;
+    getRegistryArtifactProperties(tagOrDigest: string, options?: GetRegistryArtifactPropertiesOptions): Promise<RegistryArtifactProperties>;
+    getTagProperties(tag: string, options?: GetTagPropertiesOptions): Promise<TagProperties>;
+    listRegistryArtifacts(options?: ListRegistryArtifactsOptions): PagedAsyncIterableIterator<RegistryArtifactProperties>;
```

Comment:
Just a general comment, but why do the operations on the repository client include `registry`? `listRegistryArtifacts` is listing all of the artifacts that the repository contains, right? Unless there are other types of artifacts, I wonder if we could shorten these method names, or if `registry` is actually adding value here.

Reply:
Good feedback! I was following the preview.1 API design. /cc @annelo-msft